ui/src/app/layout/topics/schema-section.component.html

```diff
@@ -15,7 +15,7 @@
                         <button appSpinnerWhile
                                 #p="appSpinnerWhile" (click)="(openDeleteConfirmDlg(deleteLatestSchemaDlg))"
                                 class="btn btn-danger ml-3"
-                                *ngIf="selectedSchemaVersion.isLatest && topicSubscribers.length === 0">
+                                *ngIf="selectedSchemaVersion.isLatest && (!(selectedEnvironment | async)?.stagingOnly || (schemaDeleteWithSub | async))">
```

Comment:
Hmm, where is the check for the subscribers length gone? This looks like, if the flag `schemaDeleteWithSub` is set to `false`, I can **never** delete a schema, even if **no** subscribers are on the topic...
Please also check if you have a UI test for this case (flag set to `false`, no subscribers, environment not `stagingOnly` - must be allowed to delete latest schema)